gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowExecutionResourceLocalHandler.java

```diff
@@ -168,6 +174,24 @@ public static FlowExecution convertFlowStatus(org.apache.gobblin.service.monitor
         .setJobStatuses(jobStatusArray);
   }
 
+  private static org.apache.gobblin.service.Issue convertIssue(Issue issues) {
```

Comment:
Add a javadoc that this is to convert a runtime issue to a gobblin service reported issue?